ts/themes/switchPrimaryColor.tsx

```diff
@@ -0,0 +1,21 @@
+import { find } from 'lodash';
+import { Dispatch } from '@reduxjs/toolkit';
+import { applyPrimaryColor } from '../state/ducks/primaryColor';
+import { COLORS, getPrimaryColors, PrimaryColorStateType } from './constants/colors';
+
+export function findPrimaryColorId(hexCode: string): PrimaryColorStateType | undefined {
+  const primaryColors = getPrimaryColors();
+  return find(primaryColors, { color: hexCode })?.id;
+}
+
```

Comment:
can you point me to which files calls this function?

Reply:
It's used in`ts/themes/switchTheme.tsx`